File: 10/10.c
#include <stdio.h>
int isNeon(int number){
        int sqnum = number*number;
        int SumofDigits=0;
        while(sqnum>0){
            SumofDigits+= sqnum%10;
            sqnum/=10;
        }
        return SumofDigits;
    }
int main() {
    int num;
    printf("Enter a number: ");
    scanf("%d",&num);
    if(isNeon(num)==num) {
            printf("Neon Number");
        }
        else {
            printf("Not a Neon Number");
        }
    return 0;
}

File: 10/10.py
def isNeon(number):
    sqnum = number**2
    SumofDigits=0
    while sqnum>0:
        SumofDigits+= sqnum%10
        sqnum//=10
    return SumofDigits
    
num = int(input("Enter a number: "))
if isNeon(num)==num:
    print("Neon Number")
else:
    print("Not a Neon Number")
